nanome/_internal/_shapes/_serialization/_anchor_serializer.py

```diff
@@ -0,0 +1,29 @@
+from nanome._internal._util._serializers import _TypeSerializer, _UnityPositionSerializer
+from nanome._internal._shapes import _Anchor
+from nanome.util.enums import ShapeType
+
+class _AnchorSerializer(_TypeSerializer):
+    def __init__(self):
+        self._offset = _UnityPositionSerializer()
+
+    def version(self):
+        return 0
+
+    def name(self):
+        return "ShapeAnchor"
+
+    def serialize(self, version, value, context):
+        context.write_long(value._target)
+        context.write_byte(int(value._anchor_type))
+        context.write_using_serializer(self._offset, value._local_offset)
+        context.write_using_serializer(self._offset, value._global_offset)
```

Comment:
Is this capturing all three values all on self._offset? I'm confused

Reply:
`self._offset` is a UnityPositionSerializer, so yes

Reply:
I totally forgot how plugin serialization worked for a sec..._offset is the serializer, not the shape property. my bad.